troubleshooting/troubleshooting-vse-path-too-long-exception.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Path Too Long Exception when Building Project in Microsoft Visual Studio
+description: AppBuilder Projects with fully qualified file names longer that 260 symbols fail to build in Visual Studio Extension.
+type: troubleshooting
+page_title: VS Extension Build Fails when File Path Length is Longer than 260 Symbols 
```

Comment:
* Capitalized When, Is and Than
* Consider whether to use Characters instead of Symbols in one of the two meta fields. Not mandatory.